packages/tokens/src/figma-tokens/_docs.json

```diff
@@ -41,13 +41,45 @@
       }
     },
     "badge": {
-      "description": {
-        "value": "Badges are used to draw attention and display statuses or counts.",
-        "type": "text"
+      "default": {
+        "description": {
+          "value": "Badges are used to draw attention and display statuses or counts.",
+          "type": "text"
+        },
+        "title": {
+          "value": "Badge",
+          "type": "text"
+        }
       },
-      "title": {
-        "value": "Badge",
-        "type": "text"
+      "variants": {
+        "title": {
+          "value": "Variants",
+          "type": "text"
+        },
+        "description": {
+          "value": "Use the pill attribute to give badges rounded edges.",
```

Comment:
I think this should be "Set the variant attribute to change the badge’s variant."

Reply:
thanks - it is resolved